src/Features/Core/Portable/SymbolMapping/SymbolMappingResult.cs

```diff
@@ -9,6 +13,9 @@ internal class SymbolMappingResult
 
         internal SymbolMappingResult(Project project, ISymbol symbol)
         {
+            Contract.ThrowIfNull(project);
+            Contract.ThrowIfNull(symbol);
```

Comment:
📝 These should be argument validations instead of `Contract.ThrowIfNull`.

Reply:
We'll be converting them all to the new syntax soon enough I suspect...